wagtaildraftail/client/wagtaildraftail.js

```diff
@@ -54,4 +64,18 @@ const initDraftailEditor = (fieldName, options = {}) => {
 
 window.initDraftailEditor = initDraftailEditor;
 
+window.wagtaildraftail = {
```

Comment:
Should this be `wagtaildraftail` or `wagtailDraftail`? The latter comes naturally to me from JS land.